src/schema/SearchableItem/SearchableItemPresenter.ts

```diff
@@ -0,0 +1,119 @@
+import moment from "moment"
+import { stripTags } from "lib/helpers"
+
+const DATE_FORMAT = "MMM Do, YYYY"
+
+export class SearchableItemPresenter {
+  private item: any
+
+  constructor(item: any) {
+    this.item = item
+  }
+
+  formattedDescription(): string | undefined {
+    const { description, display } = this.item
+
+    switch (this.item.label) {
```

Comment:
Note that this is currently missing formatting for Show and Booth types. Since that formatting is quite complicated, I'm planning to open up a separate PR to cover that.

Reply:
Cool, sounds good!